ssn/ssn_separated/ssn.ttl

```diff
@@ -433,10 +354,10 @@ ssn:BatteryLifetime rdf:type owl:Class ;
 ###  http://www.w3.org/ns/ssn/Condition
 ssn:Condition rdf:type owl:Class ;
               rdfs:subClassOf ssn:Property ;
-              rdfs:comment "Used to specify ranges for qualities that act as conditions on a system/sensor's operation.  For example, wind speed of 10-60m/s is expressed as a condition linking a quality, wind speed, a unit of measurement, metres per second, and a set of values, 10-60, and may be used as the condition on a MeasurementProperty, for example, to state that a sensor has a particular accuracy in that condition." ;
+              rdfs:comment "Used to specify ranges for qualities that act as conditions on a Systems' operation." ;
+              skos:example "For example, wind speed of 10-60m/s is expressed as a condition linking a quality, wind speed, a unit of measurement, metres per second, and a set of values, 10-60, and may be used as the Condition on a MeasurementProperty, for example, to state that a Sensor has a particular Accuracy in that condition." ;
```

Comment:
mention of unit of measurement here is confusing. Should be rewritten once ACTION-318 is completed